src/tests/JIT/opt/Casts/castclass_valuetype.il

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+.assembly extern mscorlib { auto }
+.assembly extern xunit.core {}
+.assembly extern System.Console { auto }
+.assembly extern System.Runtime { .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A ) }
+
+.assembly 'castclass_valuetype' { }
+
+.class public auto Program extends [System.Runtime]System.Object
+{
+  .method public hidebysig static int32 Main() cil managed
+  {
+    .custom instance void [xunit.core]Xunit.FactAttribute::.ctor() = (
```

Comment:
I assume the [Fact] attribute is not needed here (or then .entrypoint is not needed if it's part of the merged runtime tests)?

I'm not asking to remove it, just curious

Reply:
[Fact] is needed as JIT/opt is a merged test group.  The .entrypoint will have no impact on merged test group execution (this test is a library that is referenced by the merged test group, so this .entrypoint isn't the overall entry).

We've been keeping the .entrypoint annotations in ilproj tests.  I'm not entirely sure of the details, but my guess is that the basic wrappers (for BuildAsStandalone or RequiresProcessIsolation) have C#-specific aspects to them.

Reply:
@markples I see, thanks, I thought that .entrypoint will confuse a merged test group